client/src/app/AudioButton/AudioButton.js

```diff
@@ -10,22 +12,28 @@ AudioButton.propTypes = {
 
 export default function AudioButton({ onClick, isSongPlaying, isSmall }) {
   return (
-    <Button onClick={onClick} data-testid="audio-button" isSmall={isSmall}>
-      {isSongPlaying ? <PauseIcon title="pause" /> : <PlayIcon title="play" />}
-    </Button>
+    <motion.div variants={clickAnimation} initial="rest" whileTap="pressed">
+      <Button onClick={onClick} data-testid="audio-button" isSmall={isSmall}>
```

Comment:
Just if that matters:  You could think of placing the ``data-testid`` attribute at the end of the line, so that the other attributes can be grasp more easily, as they are maybe of a "more" informational value at this point.